libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.ts

```diff
@@ -32,6 +32,7 @@ export class ManualProjectInputComponent implements OnChanges {
     description: new UntypedFormControl(''),
     start: new UntypedFormControl(null),
     end: new UntypedFormControl(null),
+    acronym: new UntypedFormControl('', [Validators.maxLength(25)]),
```

Comment:
We do not have to be so restrictive here. Limiting it to 255 is enough.
Although it is called acronym, it is often also just a shorter name which could exceed 25 characters :)

Reply:
Done